### PR TITLE
Updated index.md with Grammar and Syntax Fixes

### PR DESCRIPTION
**Pull request by Novaenn**
Updated index.md fixing grammar mistakes and Editing Errors (adding ``` to close the coding block and fixing the variable's name and general syntax errors).

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/cplusplus/for-loop/index.md [hunk 1]
```diff
@@ -35,14 +35,14 @@ for (; a <= 10 ;)
 ```
 Though you can't ignore the condition, because empty condition is defaulted as false, so the for-loop won't run at all.
 
-### init
+### Init
```
Comment (RandellDawson) [inline guide/english/cplusplus/for-loop/index.md]: Do not change spelling here.

File: guide/english/cplusplus/for-loop/index.md [hunk 2]
```diff
@@ -35,14 +35,14 @@ for (; a <= 10 ;)
...
 This step allows you to declare and initialize any loop control variables. This step is performed first and only once.  
 Note that the variables declared in init can only be used inside the brackets of the for loop.
 
-### condition
+### Condition
```
Comment (RandellDawson) [inline guide/english/cplusplus/for-loop/index.md]: Do not change spelling here.

File: guide/english/cplusplus/for-loop/index.md [hunk 3]
```diff
@@ -35,14 +35,14 @@ for (; a <= 10 ;)
...
 Next the condition is evaluated. If it holds true, the body of the loop is executed. If it holds false, the body of the loop does not execute and flow of control jumps to the next iteration(repetition of a process).
 
-### update
+### Update
```
Comment (RandellDawson) [inline guide/english/cplusplus/for-loop/index.md]: Do not change spelling here.

File: guide/english/cplusplus/for-loop/index.md [hunk 4]
```diff
@@ -105,7 +106,7 @@ value of a: 19
 ```
 
 ## Explanation
-Here's the initialization condition is first set to a=10. The loop first checks for this condition. It then checks for the condition expression i.e a<20 which holds true as 10<20(for the first case). Now the body of the loop is executed and we get the output "Value of a: 10". Then the update expression is executed which adds the number 1 to 'a' and the value of 'a' gets updated to 11 and the same steps are followed (as above) until the value of v reaches less than 20 i.e 19.
+Here's the initialization condition which is first set to a=10. The loop first checks for this condition. It then checks for the condition expression i.e a<20 which holds true as 10<20(for the first case). Now the body of the loop is executed and we get the output "Value of a: 10". Then the update expression is executed which adds 1 to 'a' and the value of 'a' gets updated to 11 and the loop restarts with the same steps written above until the value of 'a' reaches less than 20 i.e 19.
```
Comment (RandellDawson) [inline guide/english/cplusplus/for-loop/index.md]: The current version on the master branch has changed, do you will need to resync your fork and resolve merge conflicts as some of your changes are not needed any longer.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
Please see requests for change.

**Review by cmccormack — COMMENTED** [review]
Thank you for your contribution.  At this time we have decided not to merge your pull request.  We welcome future pull requests from you.

**Explanation**
The changes in this PR conflict with the master branch.  The file edited in this commit has since been edited, moved, or deleted.  Please feel free to update your fork of the freeCodeCamp master branch and submit your edits in a new Pull Request.

Happy Coding!

**Pull request closed without merging**